sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryToTableIT.java

```diff
@@ -232,25 +232,24 @@ private void verifyStandardQueryRes() throws Exception {
 
   @Before
   public void setupBqEnvironment() {
+    Long timeSeed = System.currentTimeMillis();
+    Integer random = new Random(timeSeed).nextInt(900) + 100;
+    this.bigQueryDatasetId += (timeSeed.toString() + "_" + random.toString());
```

Comment:
Here is where I would just inline the string so you don't use `+=` to tack it on.